### PR TITLE
Updated rule list to work with current bots.

### PR DESCRIPTION
**Pull request by moebkun**
This is mainly a work-around due to current issues, but It accurately flags names, invis characters (mainly the current ones spotted most recently, but could easily be added new ones).
This should be easier to add new rules to as well.

for testing purposes remember to remove the auto-update line.

### CODE REVIEW

**Review by ClusterConsultant — COMMENTED** [review]
These changes seem to be sane and overall reasonable. The new structure is not in line with how things were previously done but I think they are more sustainable and easier to read. I'm going to submit this to testers before publishing fully, but I think this is a step in the right direction.

**Review by ClusterConsultant — APPROVED** [review]

File: staging/cfg/rules.official.json [hunk 1]
```diff
@@ -92,124 +42,20 @@
...
-			},
-			"description": "(catbot) Lag-bot",
-			"triggers": {
-				"username_text_match": {
-					"case_sensitive": false,
-					"mode": "regex",
-					"patterns": [
-						"Lag-Bot \\d+"
+						"\u0e4a",
+						"\u202c"
```
Comment (ClusterConsultant) [inline staging/cfg/rules.official.json:58]: add U+200E (Left-To-Right Mark), U+200D (Zero Width Joiner), U+200C (Zero Width Non-Joiner), U+200B (Zero Width Space), U+2060 (Word Joiner), U+202C (Pop Directional Formatting), U+FEFF (Zero Width No-Break Space), and finally the U+200F (Right-To-Left Mark)

**Review by ClusterConsultant — APPROVED** [review]
👍

**Comment by PazerOP** [comment]
Some of these changes will be later integrated into an upcoming update of the rules json format.

**Pull request closed without merging**